changes/9322.assign_wcs.rst

```diff
@@ -0,0 +1 @@
+Remove deprecated util.reproject function
```

Comment:
Do people actually read the change log? If so, might be useful to repeat the original message here on when the deprecation first happened and what the modern alternative is.

```python
        "'reproject()' has been deprecated since 1.17.2 and "
        "will be removed in a future release. "
        "Use 'stcal.alignment.util.reproject()' instead.",
```

```suggestion
Remove deprecated util.reproject function.
```